.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,37 @@
+
+name: update-version
+on:
+  pull_request:
+          types: [opened]
+          branches: [master]
+
+jobs:
+  update-version-in-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          ref: ${{ github.head_ref }}
+
+      - if: github.actor == "release-please[bot]"
+        name: find-and-replace-version
+        run: |
+          VERSION=$(cat version.txt)
+          sed -i -r 's/[0-9]+.[0-9]+.[0-9]+(-dev)?/'"$VERSION"'/' cmd/cloud_sql_proxy/cloud_sql_proxy.go
+          
+      - if: github.actor == "release-please[bot]"
```

Comment:
Is there any advantage to these being different steps?

Reply:
it's mostly for logging purposes. if something goes wrong, we know exactly which step is the problem. We can also do it in one step if you prefer that.

Reply:
I have a slight preference towards a single step for readability (and mostly because of the multiple if checks), but I haven't seen how the logging is handled. If you feel the separate steps is significant for debugging, please feel free to leave.

Reply:
Doing it in one step should be fine. I can update this.